site/docs/reference/Connectors/materialization-connectors/MySQL/singlestore-mysql.md

```diff
@@ -0,0 +1,121 @@
+| `/advanced` | Advanced Options | Options for advanced users. You should not typically need to modify these. | object |  |
+| `/advanced/sslmode` | SSL Mode | Overrides SSL connection behavior by setting the 'sslmode' parameter. | string |  |
+| `/advanced/ssl_server_ca` | SSL Server CA | Optional server certificate authority to use when connecting with custom SSL mode | string |  |
+| `/advanced/ssl_client_cert` | SSL Client Certificate | Optional client certificate to use when connecting with custom SSL mode. | string |  |
+| `/advanced/ssl_client_key` | SSL Client Key | Optional client key to use when connecting with custom SSL mode. | string |  |
+
+#### SSL Mode
+
+Possible values:
+
+- `disabled`: A plain unencrypted connection is established with the server
+- `preferred`: Only use SSL connection if the server asks for it
+- `required`: Connect using an SSL connection, but do not verify the server's
+  certificate.
+- `verify_ca`: Connect using an SSL connection, and verify the server's
+  certificate against the given SSL Server CA, but does not verify the server's
+  hostname. This option is most commonly used when connecting to an
+  IP address which does not have a hostname to be verified. When using this mode, SSL Server
+  CA must be provided.
+- `verify_identity`: Connect using an SSL connection, verify the server's
+  certificate and the server's hostname. This is the most secure option. When using this mode, SSL Server
+  CA must be provided.
+
+Optionally, SSL Client Certificate and Key can be provided if necessary to
+authorize the client.
+
+#### Bindings
+
+| Property | Title | Description | Type | Required/Default |
+| --- | --- | --- | --- | --- |
+| **`/table`** | Table | Table name to materialize to. It will be created by the connector, unless the connector has previously created it. | string | Required |
+| `/delta_updates` | Delta Update | Should updates to this table be done via delta updates. | boolean | `false` |
+
+### Sample
+
+```yaml
+materializations:
+  ${PREFIX}/${mat_name}:
+    endpoint:
+      connector:
+        image: ghcr.io/estuary/materialize-singlestore:dev
+        config:
+          database: flow
+          address: svc-abc123.aws-region.svc.singlestore.com:3306
+          password: flow
+          user: flow
+          advanced:
+            sslmode: verify_ca
+            ssl_server_ca: <singlestore-certificate>
+    bindings:
+      - resource:
+          table: ${TABLE_NAME}
+        source: ${PREFIX}/${COLLECTION_NAME}
+```
+
+## Delta updates
+
+This connector supports both standard (merge) and [delta updates](/concepts/materialization/#delta-updates).
+The default is to use standard updates.
+
+## Reserved words
+
+SingleStore uses MySQL, which has a list of reserved words that must be quoted in order to be used as an identifier.
```

Comment:
Actually let's link this one:

https://docs.singlestore.com/db/v9.0/reference/sql-reference/restricted-keywords/list-of-restricted-keywords/

I will change the code as well to add the ones here to the connector